string robot_id
string planner_id
string scenario_id
string user_id
string name
string task_id